src/oss/confluent.sh

```diff
@@ -95,6 +95,14 @@ declare -a commands=(
     "config"
 )
 
+declare -a dynamic_cli_commands=(
```

Comment:
This could include any command which is implemented by another CLI and that gets integrated here. So, it could encompass both COS and CP. Having said that we can may be call it something like delegating_commands ...